engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/ParameterTypeEditorDeterminer.scala

```diff
@@ -0,0 +1,24 @@
+package pl.touk.nussknacker.engine.definition
+
+import java.lang.reflect.Parameter
+
+import pl.touk.nussknacker.engine.api.definition.{FixedExpressionValue, FixedValuesParameterEditor, ParameterEditor}
+
+object ParameterTypeEditorDeterminer {
+  def tryDetermine(p: Parameter): Option[ParameterEditor] = {
```

Comment:
Please add all cases but `com.cronutils.model.Cron` from `pl.touk.nussknacker.ui.definition.editor.ParameterTypeEditorDeterminer` and use this class also on ui side to avoid code split on both sides

Reply:
Done